analysis_options.yaml

```diff
@@ -1,9 +1,10 @@
 include: package:flutter_lints/flutter.yaml
 
 analyzer:
-  strong-mode:
-    implicit-casts: false
-    implicit-dynamic: false
+  language:
+    strict-casts: true
+    strict-inference: true
+    strict-raw-types: true
   errors:
     close_sinks: ignore
```

Comment:
Ok, I'm kind of seeing what you mean. I'm going to remove this line, then approve this PR so it will run the checks. If the checks fail, we'll need to add that 'ignore' line back, otherwise we should be OK.

```suggestion
```